lib/web_ui/lib/src/engine/canvaskit/image.dart

```diff
@@ -304,13 +314,14 @@ class CkImage implements ui.Image, StackTraceDebugger {
       );
       bytes = skImage.readPixels(0, 0, imageInfo);
     } else {
-      final SkData skData = skImage.encodeToData(); //defaults to PNG 100%
-      // make a copy that we can return
-      bytes = Uint8List.fromList(canvasKit.getDataBytes(skData));
-      skData.delete();
+      bytes = skImage.encodeToBytes(); //defaults to PNG 100%
     }
 
-    return bytes.buffer.asByteData(0, bytes.length);
+    if (bytes != null) {
+      return bytes.buffer.asByteData(0, bytes.length);
```

Comment:
nit: instead of if/else `return bytes?.buffer......`?

Reply:
Done